ticktacotoe.sublime-project

```diff
@@ -0,0 +1,8 @@
+{
+	"folders":
+	[
+		{
+			"path": "/C/Users/Toni/Desktop/projects/ticktacotoe"
```

Comment:
You could also make this `"path": "."` which means "the directory this project file is in", which will work for other people as well (who don't have a Users/Toni directory).